test/integration/user_can_view_order_history_and_details_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.